### DISCUSSION TITLE
Question: Is there a way to directly get the lastest nightly version?

### DISCUSSION THREAD

**Discussion by serhergz** [microsoft/vscode-react-native#2214] [Q&A] [answered]
Context:
I am trying to automate testing, i would like to test against the nightly version, so that i can detect potential problems early.

Question:
Is there a way to get the latest nightly version directly from a github or other? I don't want to depend on the market place.

Thanks.

**Comment by EzioLi01** [comment, marked as answer]
Answered in related issue. Thanks!
